src/search/searchContext.js

```diff
@@ -4,6 +4,9 @@ export const searchContext = React.createContext({
   categories: [],
   category: null,
   setCategory: () => {},
-  HTTPSsupport: null, // null | true | false
+  allHTTPS: [],
```

Comment:
`allHTTPS` doesn't not change, it is merely a list of options. Remove it from the context and put it in the component directly. The context is only for application state that can change.